src/Analyzers/MSTest.Analyzers/TestMethodShouldNotBeIgnoredAnalyzer.cs

```diff
@@ -29,7 +29,8 @@ public sealed class TestMethodShouldNotBeIgnoredAnalyzer : DiagnosticAnalyzer
         Description,
         Category.Design,
         DiagnosticSeverity.Info,
-        isEnabledByDefault: true);
+        isEnabledByDefault: true,
+        disableInAllMode: true);
```

Comment:
The implementation for globalconfig generation will prevent having something enabled by default but disabled in all mode.
Because it's not super obvious to me what the behavior should be.

Maybe in this case, it should remain "info" in all mode, and not completely disabled?